destroyed_account_db.go

```diff
@@ -77,8 +77,10 @@ func (db *DestroyedAccountDB) getFirstKeyInBlock(block uint64) []byte {
 
 // GetAccountsDestroyedInRange get list of all accounts between block from and to (including from and to).
 func (db *DestroyedAccountDB) GetAccountsDestroyedInRange(from, to uint64) ([]common.Address, error) {
-	firstKey := db.getFirstKeyInBlock(from)
```

Comment:
Please remove `getFirstKeyInBlock` since there is no other uses of it. Also, please run test in Aida to make sure that priming provides correct state hashes.

Reply:
The behaviour before: `getFirstKeyInBlock` always returned nil. Therefore the retrieved range was always `0-to`. In all code occurences of priming the specified block range always starts from `0` and that's the reason, why it never broke anything. Since I tested that my change correctly converts block number it should work straight away.